blaze/server/client.py

```diff
@@ -65,13 +69,17 @@ class Client(object):
 
     blaze.server.server.Server
     """
-    __slots__ = 'url'
+    __slots__ = 'url', 'mode', 'serial'
 
-    def __init__(self, url, **kwargs):
+    def __init__(self, url, mode='json', **kwargs):
         url = url.strip('/')
         if not url[:4] == 'http':
             url = 'http://' + url
         self.url = url
+        if mode not in ('json', 'pickle'):
+            raise ValueError('mode must be either pickle or json')
```

Comment:
I wonder if this could be cheaply generalized to any serialization format by accepting dumps/loads functions in the Client/Server.  This would let us extend out to other formats like msgpack.